tests/models/x_clip/test_modeling_x_clip.py

```diff
@@ -731,3 +731,25 @@ def test_inference(self):
         expected_logits = torch.tensor([[14.0181, 20.2771, 14.4776]], device=torch_device)
 
         self.assertTrue(torch.allclose(outputs.logits_per_video, expected_logits, atol=1e-3))
+
+    @slow
+    def test_inference_interpolate_pos_encoding(self):
+        # ViT models have an `interpolate_pos_encoding` argument in their forward method,
+        # allowing to interpolate the pre-trained position embeddings in order to use
+        # the model on higher resolutions. The DINO model by Facebook AI leverages this
+        # to visualize self-attention on higher resolution images.
+        model = XCLIPModel.from_pretrained("microsoft/xclip-base-patch32").to(torch_device)
+
+        image_processor = XCLIPProcessor.from_pretrained("microsoft/xclip-base-patch32", size=480)
+
+        video = prepare_video()
+        inputs = image_processor(text="what's in the video", videos=video, return_tensors="pt").to(torch_device)
+
+        # forward pass
+        with torch.no_grad():
+            outputs = model(**inputs, interpolate_pos_encoding=True)
```

Comment:
Can you add in the test to check an error is raise if `nterpolate_pos_encoding=False`?